lib/engine_wrapper.py

```diff
@@ -593,7 +595,12 @@ def getHomemadeEngine(name: str) -> type[MinimalEngine]:
     :return: The engine with this name.
     """
     from lib import strategies
-    engine: type[MinimalEngine] = getattr(strategies, name)
+    from test_bot import strategies as test_strategies
+    engine: type[MinimalEngine]
+    if __name__ == "main":
+        engine = getattr(strategies, name)
+    else:
+        engine = getattr(test_strategies, name)
```

Comment:
This doesn't work since `__name__` for engine_wrapper.py is never `"__main__"` (or `"main"`). This breaks using homemade engines in normal use since `test_strategies` would always be used.